src/coreclr/vm/comutilnative.cpp

```diff
@@ -1309,6 +1309,61 @@ extern "C" void QCALLTYPE GCInterface_AddMemoryPressure(UINT64 bytesAllocated)
     END_QCALL;
 }
 
+extern "C" void QCALLTYPE GCInterface_EnumerateConfigurationValues(QCall::StringHandleOnStack nameHandle, QCall::StringHandleOnStack valueHandle, EnumerateConfigurationValuesCallback callback)
+{
+    QCALL_CONTRACT;
+
+    BEGIN_QCALL;
+    GCInterface::EnumerateConfigurationValues(nameHandle, valueHandle, callback);
+    END_QCALL;
+}
+
+struct ConfigurationvaluesEnumerationContext
+{
+    QCall::StringHandleOnStack* nameHandle;
+    QCall::StringHandleOnStack* valueHandle;
+    EnumerateConfigurationValuesCallback callback;
+};
+
+void ConfigurationValue(void* context, const char* name, int type, int64_t intValue, const char* stringValue, bool boolValue)
```

Comment:
```suggestion
static void ConfigurationValue(void* context, const char* name, int type, int64_t intValue, const char* stringValue, bool boolValue)
```

I'd prefer an anonymous namespace but that makes indentation troublesome.